docs/concepts/storage-profiles-content.rst

```diff
@@ -0,0 +1,22 @@
+Flocker Storage Profiles are a way to address the storage requirements for your application, enabling you to choose the best set of service attributes from your storage provider.
+
+For example, in a development environment you might only want a low cost storage option.
+However, if you need storage for a production environment, you can choose a profile with high performance and high cost options.
+
+Flocker Storage Profiles require support from your storage driver, and you are able to choose from the following profiles:
+
+* Gold: This profile is suited for applications that have high performance requirements from storage.
+  For example, databases.
+* Silver: This profile is suited for applications that might not require such high performance. 
+  Typically, this would align with default storage options.
+* Bronze: This profile is suited for applications that have no requirements for performance, and therefore a low cost option can be selected.
+
+Please be aware that the actual specification of these profiles may differ between each storage provider.
+The definition for each profile should be documented in the storage providers documentation.
+
+Currently, only a selection of :ref:`backends supported by Flocker <supported-backends>` support Flocker Storage Profiles.
+More information about support for profiles can be found in the :ref:`configuration documentation for each backend <supported-backends>`.
+
+.. note::
+	Flocker Storage Profiles is a new and exciting implementation, but it currently only has minimal features.
```

Comment:
Does "new and exciting" sound good to people? I think this was probably ripped from my original seed doc, but I wasn't sure if that sounded too tongue-in-cheek. 

Potentially something like:
`The idea of Flocker Storage Profiles is just getting started, and we hope to iterate on expanding functionality in this area. If you use a Storage Profile, it would be great to :ref:`hear from you <talk-to-us>` about how it is being used and what features you would like to see in future releases.`

Would express the same idea without the cheesiness of "new and exciting"?

Fine either way, just something I was thinking about.